Tests/FluentAssertions.Specs/Execution/TestFrameworkProviderTests.cs

```diff
@@ -59,25 +59,6 @@ public void When_running_test_with_unknown_test_framework_it_should_throw()
             .WithMessage("*the test framework 'foo' but this is not supported*");
     }
 
-    [Fact]
-    public void When_running_test_with_direct_bound_but_unavailable_test_framework_it_should_throw()
-    {
-        // Arrange
-        var configuration = new Configuration(new TestConfigurationStore())
-        {
-            TestFrameworkName = "nspec3"
-        };
-
-        var testFrameworkProvider = new TestFrameworkProvider(configuration);
-
-        // Act
-        Action act = () => testFrameworkProvider.Throw("MyMessage");
-
-        // Assert
-        act.Should().Throw<InvalidOperationException>()
-            .WithMessage("*test framework 'nspec3' but it could not be found*");
-    }
-
```

Comment:
Not sure, if this is supposed to stay in tests?

Reply:
Hmm.. test for "unsupported framework" is below that.. so technically this scenario is covered..

Reply:
This test exercises the scenario where the selected test provider does not derive from `LateBoundTestFramework`.
https://github.com/fluentassertions/fluentassertions/blob/48e5af9f3d3655ff6fb343c12256e0bc64da9d07/Src/FluentAssertions/Execution/TestFrameworkProvider.cs#L76-L78

With nspec gone `XUnit2TestFramework` is the only other `ITestFramework` that does not derive from `LateBoundTestFramework` and can be unavailable.

If we still want to exercise that path it would have to be from another project that doesn't load xunit.
I'm fine not covering that path and get on with this PR.